src/modules/tests/candidate/__tests__/candidate-selector.spec.ts

```diff
@@ -41,4 +51,26 @@ describe('candidate selector', () => {
       expect(formatDriverNumber('ABCDE123456Z78YX')).toBe('ABCDE 123456 Z78YX');
     });
   });
+
+  describe('getPostalAddress', () => {
+    it('should output the address', () => {
+      expect(getPostalAddress(candidate).addressLine1).toEqual('1 Example Street');
```

Comment:
Could this test be simplified to comparing a higher level reference?
```
expect(getPostalAddress(candidate)).toEqual(candidate.candidateAddress);
```

Reply:
fixed